.github/workflows/PR_summary.yml

```
@@ -99,6 +99,9 @@ jobs:
         ## Technical debt changes
         techDebtVar="$(./scripts/technical-debt-metrics.sh pr_summary)"
 
-        message="$(printf '%s [%s](%s)%s\n\n%s\n\n---\n\n%s\n\n---\n\n%s\n' "${title}" "$(git rev-parse --short HEAD)" "${hashURL}" "${high_percentages}" "${importCount}" "${declDiff}" "${techDebtVar}")"
+        # store in a file, to avoid "long arguments" error.
+        printf '%s [%s](%s)%s\n\n%s\n\n---\n\n%s\n\n---\n\n%s\n' "${title}" "$(git rev-parse --short HEAD)" "${hashURL}" "${high_percentages}" "${importCount}" "${declDiff}" "${techDebtVar}" |
+          jq -R -n -c '{"body": [inputs]}' > messageFile.json
```

I am trying here to produce a json file of the form `{"body": "whatever_the_printf_outputs"}`.  Of course, whatever is output by `printf` needs to be escaped as well.  Unfortunately, I do not know how to do that!

---

I think the [`--slurp`](https://jqlang.github.io/jq/manual/#invoking-jq) command line option might work here?
```suggestion
        printf '%s [%s](%s)%s\n\n%s\n\n---\n\n%s\n\n---\n\n%s\n' "${title}" "$(git rev-parse --short HEAD)" "${hashURL}" "${high_percentages}" "${importCount}" "${declDiff}" "${techDebtVar}" |
          jq -Rs -n -c '{"body": [inputs]}' > messageFile.json
```
Only did some very crude testing locally, but the resulting JSON file did seem to have just one big string instead of one array per line.

---

Let's try this!